react-training/src/app/screens/Game/Square/styles.scss

```diff
@@ -0,0 +1,22 @@
+.square {
+  background: #fff;
+  border: 1px solid #999;
```

Comment:
Move this 2 colors to `src/scss/variables/_colors.scss`, import them and use here